slices/nodejs.yaml

```diff
@@ -0,0 +1,12 @@
+package: nodejs
+
+slices:
+  bins:
+    essential:
+      - libnode108_libs
+      - node-cjs-module-lexer_libs
+      - node-acorn_libs
+      - node-undici_libs
```

Comment:
right, so these are the dependencies I flagged missing, above in https://github.com/canonical/chisel-releases/pull/40/files#diff-142d24a2227a0d5f6f093889d9e9eed94ec886e90242dbf2c7c56bc391775959. 

Why have you included them here instead of `libnode108`? According to the deb, they should come with the latter and not `nodejs`.

Reply:
Done